styles/_header.scss

```diff
@@ -72,42 +96,123 @@ $header-height: 71px;
       color: $link_color;
       text-decoration: underline;
     }
-
-    &.sign-in { display: inline-block; }
   }
+}
 
-  .hide-on-mobile {
-    @include mobile {
-      @include visually-hidden;
-    }
-  }
+.nav-wrapper-mobile {
+  @include desktop { display: none; }
 
-  .menu-button {
-    @include tablet { display: none; }
+  .menu-button-mobile {
     background: none;
     border: 0;
-    color: $link_color;
-    display: inline-block;
-    margin-right: 10px;
-    padding: 0;
     width: auto;
+    min-width: 71px;
+    cursor: pointer;
 
     .icon-menu {
+      padding: 7px;
       vertical-align: middle;
-      width: 13px;
-      height: 13px;
+      width: 30px;
+      height: 30px;
     }
 
-    [dir="rtl"] & {
-      margin-left: 10px;
-      margin-right: 0;
+    &[aria-expanded="true"] {
+      .icon-menu {
+        background: rgba(31, 115, 183, 0.2);
+        border-radius: 50%;
+      }
     }
+  }
 
-    &:hover,
-    &:focus,
-    &:active {
-      background-color: transparent;
-      color: $link_color;
+  .menu-list-mobile {
+    position: absolute;
+    background-color: #fff;
+    box-shadow: 0 10px 10px 0 rgba(0, 0, 0, .15);
+    border-top: solid 0.1px #ddd;
+    border-bottom: solid 1px #ddd;
+    right: 0;
+    left: 0;
+    top: $header-height;
+    z-index: 2;
+
+    &[aria-expanded="false"] {
+      display: none;
+    }
+
+    &[aria-expanded="true"] {
+      display: block;
+    }
+  }
+
+  .menu-list-mobile-items {
+    .item {
+      margin: 4px 0;
+    }
+
+    li:empty:not(.nav-divider){
+      display: none;
+    }
+
+    .nav-divider {
+      border-bottom: 0.1px solid #ddd;
+      padding: 0;
+
+      &:last-child {
+        display: none;
+      }
+    }
+
+    button {
+      background: none;
+      border: none;
+      padding: 8px 24px;
+
+      width: 100%;
+      height: 100%;
+      color: $text_color;
+      cursor: pointer;
+      text-align: start;
+
+      &:active, &:focus, &:hover {
+        background-color: rgba(31, 115, 183, 0.1);
```

Comment:
We've been using the `#f3f3f3` color when hovering [dropdown items](https://github.com/zendesk/copenhagen_theme/blob/master/styles/_dropdowns.scss#L77). Example:

<img width="231" alt="Screenshot 2021-12-08 at 14 48 11" src="https://user-images.githubusercontent.com/1172767/145219263-aa32c9e4-822b-49a0-960a-0308e950d618.png">

Using a different color here will make them inconsistent.

Another issue is that, by changing from a neutral color (gray) to a shade of blue, it may not play well with themes that are branded with different colors.

To unblock this PR, I recommend using `#f3f3f3` here and add this change in a separate PR. Meanwhile, let's check with design if:
- if we should use a neutral color here or if this color should be part of the theme settings;
- this change should be applied to all dropdowns in the theme;

Reply:
Good catch, I've forwarded this comment to design